docs/api.rst

```diff
@@ -263,6 +263,7 @@ Wflow workflows
    workflows.lai_from_lulc_mapping
    workflows.add_paddy_to_landuse
    workflows.ksathorfrac
+   workflows.ksatver_vegetation
```

Comment:
Yes this is what I meant by adding the workflow function in the API docs :) the ``setup_ksatver_vegetation`` under Wflow is missing though